metricbeat/module/kubernetes/event/types.go

```diff
@@ -0,0 +1,46 @@
+package event
+
+import "time"
+
+type ObjectMeta struct {
+	Annotations       map[string]string `json:"annotations"`
+	CreationTimestamp string            `json:"creationTimestamp"`
+	DeletionTimestamp string            `json:"deletionTimestamp"`
+	GenerateName      string            `json:"generateName"`
+	Labels            map[string]string `json:"labels"`
+	Name              string            `json:"name"`
+	Namespace         string            `json:"namespace"`
+	OwnerReferences   []struct {
+		APIVersion string `json:"apiVersion"`
+		Controller bool   `json:"controller"`
+		Kind       string `json:"kind"`
+		Name       string `json:"name"`
+		UID        string `json:"uid"`
+	} `json:"ownerReferences"`
+	ResourceVersion string `json:"resourceVersion"`
+	SelfLink        string `json:"selfLink"`
+	UID             string `json:"uid"`
+}
+
+type Event struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Event should have comment or be unexported